service/src/main/kotlin/io/provenance/explorer/service/TransactionService.kt

```diff
@@ -122,6 +122,7 @@ class TransactionService(
             TxCacheRecord.findByEntityId(curr.id)!!
         } else curr
 
+    // Is this related?  What is the protoPrinter?
```

Comment:
This is where we are seeing it. The specific use case is when mapping the metadata msgs, due to how the scope/record/session/spec addresses are handled. They arent actually strings, they are `bytes` which get handled differently, with no option to decode to UTF8